aeron-driver/src/main/c/aeron_name_resolver_driver.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2014-2020 Real Logic Limited.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef AERON_NAME_RESOLVER_DRIVER_H
+#define AERON_NAME_RESOLVER_DRIVER_H
+
+#define AERON_NAME_RESOLVER_DRIVER_SELF_RESOLUTION_INTERVAL_MS (1 * INT64_C(1000))
+#define AERON_NAME_RESOLVER_DRIVER_NEIGHBOUR_RESOLUTION_INTERVAL_MS (2 * INT64_C(2000))
+#define AERON_NAME_RESOLVER_DRIVER_TIMEOUT_MS (10 * INT64_C(1000))
+#define AERON_COUNTER_NAME_RESOLVER_NEIGHBORS_COUNTER_TYPE_ID (14)
+#define AERON_COUNTER_NAME_RESOLVER_CACHE_ENTRIES_COUNTER_TYPE_ID (15)
+
+int aeron_name_resolver_driver_set_resolution_header(
```

Comment:
shouldn't the name be "aeron_driver_name_resolver".... seems odd to have "driver" at the end.

Reply:
I did tie myself in a few knots with naming conventions.  There are a couple of knock effects with changing that.  I have `aeron_name_resolver_default` and `aeron_name_resolver_csv_table`, which should also change to `aeron_default_name_resolver` and `aeron_csv_table_name_resolver` respectively.

Reply:
But I agree, will make the change and update the PR.

Reply:
I'm also tempted to rename the `aeron_name_resolver_driver_cache` to `aeron_name_resolver_cache`.  It's decoupled from the driver gossip and could potentially be a cache for any name resolver.

Reply:
Kinda what I went for in the Java version. Wanted to bring it out separately.... so, the naming would make sense to me.

Naming is hard.